iothub/device/src/Transport/AmqpIot/AmqpIotReceivingLink.cs

```diff
@@ -218,9 +218,7 @@ private void OnMethodReceived(AmqpMessage amqpMessage)
 
             try
             {
-                MethodRequestInternal methodRequestInternal = AmqpIotMessageConverter.ConstructMethodRequestFromAmqpMessage(
-                    amqpMessage,
-                    new CancellationToken(false));
```

Comment:
This was the only spot we created the MethodRequestInternal from AMQP. Note that this token would never be cancelled